redis/commands/search/querystring.py

```diff
@@ -132,6 +132,14 @@ def __init__(self, lon, lat, radius, unit="km"):
         self.radius = radius
         self.unit = unit
 
+    def to_string(self):
+        return "[{0} {1} {2} {3}]".format(
+            self.lon,
+            self.lat,
+            self.radius,
+            self.unit
+        )
```

Comment:
This would be more succinct as an f-string:
```suggestion
        return f"[{self.lon} {self.lat} {self.radius} {self.unit}]"
```